### PR TITLE
New method applyOptions()

### PR DESCRIPTION
**Pull request by damianoporta**

### CODE REVIEW

File: src/Query.php [hunk 1]
```diff
@@ -372,6 +425,7 @@ public function compileQuery()
 
         if ($this->_parts['query'] !== null) {
             $filteredQuery->setQuery($this->_parts['query']);
+			$this->_elasticQuery->setQuery($filteredQuery);
```
Comment (markstory) [inline src/Query.php]: Indentation looks off.

Comment (damianoporta) [inline src/Query.php]: @markstory done

**Comment by markstory** [comment]
Looks good to me.

**Comment by damianoporta** [comment]
@markstory why do you want to posticipate the merging?

**Comment by markstory** [comment]
I was waiting to see if @lorenzo had any input.

**Comment by damianoporta** [comment]
ok, it is important because i also have done the paginator and it uses that method. I will sent it with another PR

**Comment by lorenzo** [comment]
Thanks @damianoporta :)

**Pull request merged**